pkg/configuration/configuration.go

```diff
@@ -54,6 +54,12 @@ const (
 	// defaultCheRequired is the default value for che.required param
 	defaultCheRequired = false
 
+	// varCheUserDeletionEnabled is a way to turn the Che user deletion logic on/off
+	varCheUserDeletionEnabled = "che.user.deletion.enabled"
+
+	// defaultCheUserDeletionEnabled is the default value for che.required param
+	defaultCheUserDeletionEnabled = false
```

Comment:
It means we have to set this property to `true` in all our clusters first before we can merge this PR, right?

Reply:
Yes, that's what I was trying to make clear with the 🚨  IMPORTANT 🚨  message at the top of the description but I guess that didn't work 😄 . Alternatively, we could make the default `true`.

Reply:
Haha. I missed that :)